### PR TITLE
Move helm charts to root & Helm release workflow

### PR DESCRIPTION
**Pull request by daniel-spray**
#### What does this PR do?
Moves charts to root directory so the Helm install command will work and adds a helm release GH action workflow which handles Helm chart versioning
#### Where should the reviewer start?
workflows
#### Any background context you want to provide?
Part of Helm support work
#### Developer Done List

- [ ] Tests Added/Updated
- [ ] Updated README.md
- [ ] Verified backward compatible
- [ ] Verified database migrations will not be catastrophic
- [x] Considered Security, Availability and Confidentiality

#### Reviewer/Approval Done List

- [ ] Tests Pass Locally
- [ ] CI Build Passes
- [ ] Verified README.md is updated
- [ ] Verified changes are backward compatible
- [ ] Reviewed impact to Security, Availability and Confidentiality (if issue found, add comments and request changes)

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -33,6 +33,26 @@ jobs:
         if: github.ref == 'refs/heads/beta'
         run: make container-build-beta
 
+  helmRelease:
```
Comment (rrobb6) [inline .github/workflows/deploy.yml:36]: Is this intended to run on the master/beta branches?

**Review by rrobb6 — COMMENTED** [review]
QQ about the helm release workflow

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -33,6 +33,27 @@ jobs:
...
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.1.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
   slackFinish:
     name: Notify Finish
     needs: deploy
```
Comment (rrobb6) [inline .github/workflows/deploy.yml:59]: Should this be updated to need "helmRelease" instead?

Comment (daniel-spray) [reply .github/workflows/deploy.yml]: Good catch 👍

**Review by rrobb6 — APPROVED** [review]
LGTM 🥇

**Pull request merged**